File: test/fixtures/elixir/multiple_d_post.ex
response = HTTPoison.post!(
  "http://localhost:28139/webservices/rest.php",
  "version=1.2&auth_user=fdgxf&auth_pwd=oxfdscds&json_data={ \"operation\": \"core/get\", \"class\": \"Software\", \"key\": \"key\" }",
  [
    {"Content-Type", "application/x-www-form-urlencoded"}
  ]
)

File: test/fixtures/python/multiple_d_post.py
import requests

headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
}

data = 'version=1.2&auth_user=fdgxf&auth_pwd=oxfdscds&json_data={ "operation": "core/get", "class": "Software", "key": "key" }'

response = requests.post('http://localhost:28139/webservices/rest.php', headers=headers, data=data)
